userspace/av-gui/av_gui/avd_client.py

```
@@ -0,0 +1,99 @@
+
+def status():
+    """Returns a dict: uptime_secs, rules_loaded, fuzzy_corpus_count,
+    tlsh_corpus_count, scan_queue_len, scan_threads (all int)."""
+    rows = _parse_rows(_request("STATUS"))
+    if not rows:
+        raise AvdError("STATUS returned no data")
+    keys = [
+        "uptime_secs", "rules_loaded", "fuzzy_corpus_count",
+        "tlsh_corpus_count", "scan_queue_len", "scan_threads",
+    ]
+    return dict(zip(keys, (int(v) for v in rows[0])))
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)